apps/web/src/components/Layout/MaintenanceBanner.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import isAfter from "date-fns/isAfter";
```

Comment:
You've got a linting warning here.  See a similar override in packages/date-helpers/src/index.ts.

Reply:
Perfect thanks, fixed by [88f4c06](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7832/commits/88f4c06c3867f6804a4f87b8a770c5ce0b996bfd)